src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -227,6 +228,64 @@ export class MoveToLineEndTerminalAction extends BaseSendTextTerminalAction {
 	}
 }
 
+export class SendSequenceTerminalCommand extends Command {
+	public static readonly ID = TERMINAL_COMMAND_ID.SEND_SEQUENCE;
+	public static readonly LABEL = nls.localize('workbench.action.terminal.sendSequence', "Send Custom Sequence To Terminal");
+
+	public runCommand(accessor: ServicesAccessor, args: any): void {
+		const terminalInstance = accessor.get(ITerminalService).getActiveInstance();
+		if (!terminalInstance) {
+			return;
+		}
+		const text = args.text.split(' ');
```

Comment:
Can `args` be a `string`, rather than a `{ text: string }`?

Reply:
Yes, `args` can contain the direct sequence. But most of the time, args are passed as `{key: val}`.

Reply:
Let's stick with `{ text: string }`, all the examples using args seem to use that 👍